tests/acceptance/ember-community-survey-2020-test.js

```diff
@@ -28,13 +28,7 @@ module('Acceptance | ember-community-survey-2020', function (hooks) {
   test('Accessibility audit', async function (assert) {
     await visit('/ember-community-survey-2020');
     await waitUntilAllChartsAreDrawn();
-    await a11yAudit({
-      rules: {
-        'heading-order': {
-          enabled: false,
-        },
-      },
-    });
```

Comment:
Note that we no longer need to make an exception to accessibility check in survey pages for 2020, 2018, and 2017.